week2/homework/src/index.js

```diff
@@ -1,3 +1,90 @@
-'use strict';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+
+const toDoList = (fs.readFileSync('list.txt', 'utf8')).split('\n');
```

Comment:
The `split` function doesnt do much since you are saving all the words with commas.

Reply:
Thanks, good call!